packages/runtime/src/widgets/TabBar.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { WidgetRegistry } from "../registry";
+import { EnsembleRuntime } from "../runtime";
+import { Tabs, ConfigProvider } from "antd";
+import { Widget } from "framework";
+import { IconProps } from "../util/types";
+import { Icon } from "./Icon";
+const { TabPane } = Tabs;
+
+export type items = {
+  label: string;
+  icon?: IconProps;
```

Comment:
I don't think icon is part of the schema?

Reply:
schema has icon: hasIcon
"properties": {
              "label": {
                "type": "string",
                "description": "Setting the tab label"
              },
              "icon": {
                "$ref": "#/$defs/HasIcon"
              },
              "widget": {
                "$ref": "#/$defs/Widget",
                "description": "Return an inline widget or specify a custom widget to be rendered as this tab's content"
              }
            }

Reply:
You're right! My mistake